scripts/ff_ci_pr_build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
That sounds reasonable. Though I should add the URL will be the same one we are downloading from.

The original script has comments on key steps and also provides help through the CLI. That said, there are no docstrings on the functions and there is no file level docstring. Feel free to clarify what additional info you would like to see over there.